web/sites/default/config/block.block.views_block__group_notifications_block_1.yml

```diff
@@ -29,3 +30,13 @@ visibility:
     pages: "/group/*\r\n/group/*/*"
     negate: false
     context_mapping: {  }
+  'entity_bundle:group':
```

Comment:
Could you talk me through why we would need both the page visibility and the entity bundle setting? It seems like it should be one or the other.

Reply:
This addition was part of previous exploration. I've removed it in my last commit.